demonstrations/learning2learn.py

```diff
@@ -0,0 +1,1205 @@
+r"""
+Learning to Learn with Quantum NN via Classical NN
+==================================================
+*Author: Stefano Mangini (mangini.stfn@gmail.com)*
```

Comment:
```suggestion
*Author: Stefano Mangini (mangini.stfn@gmail.com). Posted: XX Feb 2021. Last updated: XX Feb 2021.*
```
@stfnmangini When everything's ready to go, please modify the two dates here to the merge date :+1: Thanks!

Reply:
Will do! Thanks :)